src/components/dev-hub/project-sign-up-form.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react';
+import Modal from './modal';
+import styled from '@emotion/styled';
+import { size } from './theme';
+import { H5 } from './text';
+import Input from './input';
+import Button from './button';
+import TextArea from './text-area';
+
+const ModalContainer = styled('div')`
+    padding: ${size.default};
```

Comment:
The padding seems a bit larger than the mockups, I think the modal itself adds the padding we need? Perhaps this can just be removed now